src/Rings/groebner.jl

```diff
@@ -1,7 +1,8 @@
 export  reduce, reduce_with_quotients, reduce_with_quotients_and_unit, f4, fglm,
 		standard_basis, groebner_basis, standard_basis_with_transformation_matrix,
 		groebner_basis_with_transformation_matrix,
-		leading_ideal, syzygy_generators, is_standard_basis, is_groebner_basis
+	leading_ideal, syzygy_generators, is_standard_basis, is_groebner_basis,
+  groebner_basis_hilbert_driven
```

Comment:
Please don't use tabs (ideally configure your editor to not use tabs in the OSCAR code base -- e.g. by configuring to honor your `.editorconfig` file, see https://editorconfig.org).
```suggestion
		leading_ideal, syzygy_generators, is_standard_basis, is_groebner_basis,
		groebner_basis_hilbert_driven
```